kong/core/plugins_iterator.lua

```diff
@@ -41,7 +41,8 @@ local function load_plugin_configuration(api_id, consumer_id, plugin_name)
                                            load_plugin_into_memory,
                                            api_id, consumer_id, plugin_name)
   if err then
-    responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+    ngx.ctx.delay_response = false
```

Comment:
Double checked that this is the only place in which we call `responses.send()` in between the `ctx.delay_response = true` assignment and the response flushing - it should be safe.